src/settings/VoucherNumber/SettingsVoucherNumber.js

```diff
@@ -20,27 +22,47 @@ import SettingsVoucherNumberForm from './SettingsVoucherNumberForm';
 class SettingsVoucherNumber extends Component {
   static manifest = Object.freeze({
     voucherNumber: VOUCHER_NUMBER_START,
+    sequenceNumber: {},
   });
 
   static propTypes = {
     label: PropTypes.node.isRequired,
     stripes: stripesShape.isRequired,
     mutator: PropTypes.object.isRequired,
-    resources: PropTypes.object,
+    resources: PropTypes.object.isRequired,
+  };
+
+  state = {
+    sequenceNumber: 1,
```

Comment:
why you've introduced the local state if you previously handled change in local resource?

Reply:
I set in local state default value and call `mutator.sequenceNumber.replace(value);` only in reset case. This need for create correct path in POST request, because by default the local resource stored as an object

Reply:
hold on, `path` in resource doesn't depend on state var here. Am I wrong?

Reply:
For reset start number I need get this number from this state or from get `VOUCHER_NUMBER_START` request, but if I use value from request I need use `componentDidUpdate`

Reply:
I don't think you have to use `componentDidUpdate`. What for? As you mentioned above, you make `mutator.sequenceNumber.replace(value);` before reset. So I don't see any reason to use `componentDidUpdate`. All I'm trying to say - let's do the simplest to read solution. Store value in a local resource fits well. Making 2 sources of truth - makes it harder to understand, since you still have to use local resource to call the api.

Reply:
I changed to initialValue.